src/libs/print_settings.c

```diff
@@ -1495,7 +1570,7 @@ void *legacy_params(dt_lib_module_t *self, const void *const old_params, const s
     }
 
     // in theory pprofile can't be srgb or adobergb, but checking for them won't hurt
-    if(*pprofile == '\0' || !g_strcmp0(pprofile, "none"))
+    if(*pprofile == '\0' || !g_strcmp0(pprofile, _("none")))
```

Comment:
This will be a new translation string, which is not guaranteed to be translated exactly as the string you are trying to match. In other words: how is this not broken?

Reply:
This is in legacy param and the value was stored as _("none") so we need to check for this value here. It has been discussed with houz, see comment above.